### DISCUSSION TITLE
倍速播放

### DISCUSSION THREAD

**Discussion by LinChenMax** [ywmoyue/biliuwp-lite#726] [feature]
### 问题描述

把播放里的倍速功能优化一下，感谢

### 描述解决方案

可不可以把倍速改成和网页类似的，点击向右键快进，长按倍速

### 代码贡献

否

**Comment by ywmoyue** [comment]
区分点击和长按的还没做

**Reply by LinChenMax** [reply]
辛苦辛苦  O(∩_∩)O

**Reply by LinChenMax** [reply]
才过了两星期就完成了，感谢重视。
作者好给力🥰

**Reply by lVentus** [reply]
倍速播放开始和结束时，视频时流畅切换的，但音频会卡一下耶，是我的问题还是还没完善呀

**Reply by LinChenMax** [reply]
是的我也发现了，但是凑合能用，影响不大
加油，永远支持
